UnityProject/Assets/Scripts/Hacking/HackingProcesses/HackingProcessDoorSimple.cs

```diff
@@ -125,8 +191,10 @@ public void Shuffle(List<HackingNodeInfo> list)
 
 	public override void ServerGenerateNodesFromNodeInfo()
 	{
+		if (serverEndRoundHackingResetSetup == false) ServerRegisterHackingReset();
```

Comment:
found it. Do it inside brackets and in a new line so it is easier to see at a glance.